runtime/jnichk/jnicmem.c

```diff
@@ -122,6 +126,9 @@ jniCheckMemoryInit(J9JavaVM* javaVM)
 	PORT_ACCESS_FROM_JAVAVM(javaVM);
 
 	omrthread_monitor_t globalMonitor = omrthread_global_monitor();
+	if (0 == (javaVM->checkJNIData.options & JNICHK_NONFATAL)) {
```

Comment:
Please use the any bits macro here.

Reply:
Yeah, replace it with `J9_ARE_NO_BITS_SET`.
Please have another look.